cmd/rokka/cli/sourceimages.go

```diff
@@ -68,6 +70,38 @@ func restoreSourceImage(c *rokka.Client, args []string) (interface{}, error) {
 
 func copySourceImage(c *rokka.Client, args []string) (interface{}, error) {
 	return nil, c.CopySourceImage(args[0], args[1], args[2])
+
+}
+
+func copyAllSourceImage(c *rokka.Client, args []string) (interface{}, error) {
+
+	images := make(chan string)
+	results := make(chan copyall.CopyResult)
+	quit := make(chan bool)
+
+	copyAllOptions.SourceOrganization = args[0]
+	copyAllOptions.DestinationOrganization = args[1]
+
+	copyall.StartWorkers(copyAllOptions, rokkaClient, images, results, quit)
+
+	// Scan folders and files
+	go copyall.Scan(copyAllOptions, rokkaClient, images, quit)
+
+	// Collect results and display progress
+
+	counter := 0
+	for result := range results {
+		if result.Error != nil {
+			logger.Errorf("Copy failed for %s! %s\n", result.RokkaHash, result.Error)
+		} else {
+			logger.Printf("Copied %s\n", result.RokkaHash)
+			counter++
+		}
```

Comment:
I think both logs should use `Errorf` because that goes to `StdErr`. StdOut should be used for things which can be piped to the next program.

Reply:
actually maybe it would make more sense to just display a progress "bar" (would need to look it up how to do that easily) and store the successful/erroring images in a slice. Then we could in the end return that as the result and display a table (or so) for the erroring output.

Reply:
I'll give this a try 
https://github.com/cheggaaa/pb